<!DOCTYPE html>
<html><head><title>Catch</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">3 references to Catch</div><div class="rA">Assembly-CSharp (3)</div><div class="rG" id="Assembly-CSharp"><div class="rF"><div class="rN">Assets\UnitTests\Tools\ChainingAssertion.Unity.cs (1)</div>
<a href="../Assets/UnitTests/Tools/ChainingAssertion.Unity.cs.html#179"><b>179</b>var exception = <i>Catch</i>&lt;T&gt;(testCode, message);</a>
</div>
<div class="rF"><div class="rN">Assets\UnitTests\UnitTests.cs (2)</div>
<a href="../Assets/UnitTests/UnitTests.cs.html#986"><b>986</b>AssertEx.<i>Catch</i>&lt;InvalidOperationException&gt;(() =&gt; d.Disposable = id2);</a>
<a href="../Assets/UnitTests/UnitTests.cs.html#4303"><b>4303</b>AssertEx.<i>Catch</i>&lt;TimeoutException&gt;(() =&gt;</a>
</div>
</div>
</body></html>